app/lib/features/onboarding/pages/link_email_page.dart

```diff
@@ -100,12 +99,11 @@ class LinkEmailPage extends ConsumerWidget {
 
   Future<void> linkEmail(BuildContext context, WidgetRef ref) async {
     if (!formKey.currentState!.validate()) return;
-    final client = ref.read(alwaysClientProvider);
-    final manager = client.threePidManager();
+    final account = ref.read(accountProvider);
     EasyLoading.show(status: L10n.of(context).linkingEmailAddress);
     try {
-      await manager.requestTokenViaEmail(emailController.text.trim());
-      ref.invalidate(emailAddressesProvider);
```

Comment:
will this be triggered automatically now? Or why are you removing this `ref.invalidate`?

Reply:
@bitfriend only this one remaining. ...

Reply:
Sorry for my bad.
I confused it with [this PR](https://github.com/acterglobal/a3/pull/1598).
Fixed by https://github.com/acterglobal/a3/pull/1896/commits/9f5f5be142117539be41c6600db28ab5952e6fd8